tests/source/expr.rs

```diff
@@ -251,7 +251,7 @@ fn repeats() {
 
 fn blocks() {
     if 1 + 1 == 2 {
-        println!("yay arithmetix!");
```

Comment:
that was intentionally misspelled :)
but you can keep the change if that's easier, because it really doesn't matter